packages/js/components/src/plugins/test/index.js

```diff
@@ -4,16 +4,27 @@
 import { render, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { createElement } from '@wordpress/element';
+import { useDispatch } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 
-import { Plugins } from '../index.js';
+import { Plugins } from '../index';
+
+jest.mock( '@wordpress/data', () => ( {
```

Comment:
had to adjust mocking for useSelect and useDispatch now there is no HOC to pass install function to.